src/main/java/use_case/logout/LogoutInputData.java

```diff
@@ -0,0 +1,15 @@
+package use_case.logout;
+
+public class LogoutInputData {
+
+    private final String email;
```

Comment:
this attribute is not necessary. its also not neccessary in the output boundary so remove that as well